jigglypuffRL/nn/loss.py

```diff
@@ -0,0 +1,85 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+class TDLoss(nn.MSELoss):
+    def __init__(self, actor, critic, target_critic, stochastic=False, 
+                 prioritized_replay_buffer_weight=None, alpha=None,
+                 *args, **kwargs):
+        super(TDLoss, self).__init__(args, kwargs)
+        self.actor = actor
+        self.critic = critic
+        self.target_critic = target_critic
+        self.stochastic = stochastic
+        self.prioritized_replay_buffer_weight # Only used in DQN
```

Comment:
initialize this to `prioritized_replay_buffer_weight`

Reply:
Oo, missed that